rps.js

```diff
@@ -0,0 +1,15 @@
+const Playgame = function(userpicked) {
+
+    let random = Math.random();
+    if (random <= 0.33)
+        computerpick = 'Rock'
+    if (random > 0.33 && random <= 0.66)
+        computerpick = 'Paper'
+    if (random > 0.66)
+        computerpick = 'Scissors'
+
+    if (userpicked === computerpick)
```

Comment:
you did not include the proper possible outcomes for this game 

include situations with different inputs and different possible outcomes 

the code now will only return win if the userpick is the same as computerpick